pep-0480.txt

```diff
@@ -4,10 +4,10 @@ Version: $Revision$
 Last-Modified: $Date$
 Author: Trishank Karthik Kuppusamy <karthik@trishank.com>,
         Vladimir Diaz <vladimir.diaz@nyu.edu>,
-        Justin Cappos <jcappos@nyu.edu>
-BDFL-Delegate: Richard Jones <r1chardj0n3s@gmail.com>
-Discussions-To: DistUtils mailing list <distutils-sig@python.org>
-Status: Deferred
+        Justin Cappos <jcappos@nyu.edu>, Marina Moore <mm9693@nyu.edu>
+BDFL-Delegate: Paul Moore <p.f.moore@gmail.com>
```

Comment:
This feels more like a Warehouse PEP than an interoperability one, and I'm not at all familiar with it, so @dstufft is more likely to be the correct BDFL-Delegate for it.

If I were to take it, I'd probably simply look for someone with more familiarity than I have with the subject to pick it up anyway - and in that case Donald would be my first choice 🙂

Reply:
OK! I'll switch it.